src/Webapi/Webapi__WorkerGlobalScope.res

```diff
@@ -0,0 +1,71 @@
+/**
+ * Spec: https://html.spec.whatwg.org/multipage/workers.html#workerlocation
+ *
+ * Unlike Webapi__Dom__Location, this one is readonly
+ */
+type workerLocation = {
+  href: string,
+  protocol: string,
+  host: string,
+  hostname: string,
+  port: string,
+  pathname: string,
+  search: string,
+  hash: string,
+  origin: string,
+}
+
+/**
+ * Spec: https://html.spec.whatwg.org/multipage/workers.html#workerglobalscope
+ */
+module Impl = (
+  T: {
+    type t
+  },
+) => {
+  type t_workerGlobalScope = T.t
+
+  @val external self: t_workerGlobalScope = "self"
+
+  @get external location: t_workerGlobalScope => workerLocation = "location"
+
+  @get external navigator: t_workerGlobalScope => Webapi__WorkerNavigator.t = "navigator"
+
+  @variadic @val
+  external importScripts: array<string> => unit = "importScripts"
+
+  type eventHandler = Dom.event => unit
+  @get external onerror: t_workerGlobalScope => eventHandler = "onerror"
```

Comment:
Forgot about this -- these events (minus the error event) do not have phantom types in `Dom.ml`, gonna add module stubs for them